_data/team.yml

```diff
@@ -51,9 +51,9 @@
   name: Steve Lydford
   image: /assets/img/custom/team/steve_lydford.jpg
   role: Software Craftsman
-  shortDescription: Steve is a software craftsman and author with over 15 year’s professional experience. He first discovered his passion for code at the age of seven, when his father brought home a Sinclair Spectrum ...
+  shortDescription: Steve is a software craftsman and author with over 15 years professional experience. He first discovered his passion for code at the age of seven, when his father brought home a Sinclair Spectrum ...
```

Comment:
This website seems to disagree: http://www.businesswritingblog.com/business_writing/2008/08/years-or-years.html

> But why use the apostrophe? Because years is a possessive form.
> 
> Examples:
> 
> The pay of a week is a week's pay.
> The sabbatical of a year is a year's sabbatical.
> The experience of five years is five years' experience.
> 
> Many people don't like the "years' experience" construction, and I don't blame them. It's odd. But it is also correct.